app/processing/statement/get-statement.js

```diff
@@ -0,0 +1,26 @@
+const { getDetails, getAddress, getDetailedFunding, getScheme, getDetailedPayments } = require('./components')
+const getCalculation = require('../calculation')
+const getPaymentRequest = require('../payment-request')
+const getSettlement = require('../settlement')
+
+const getStatement = async (settlementId) => {
+  const settlement = await getSettlement(settlementId)
+  const paymentRequestId = settlement.paymentRequestId
+  const paymentRequest = await getPaymentRequest(paymentRequestId)
+  const calculation = await getCalculation(paymentRequestId)
+  const details = await getDetails(calculation.sbi)
+  const address = await getAddress(calculation.sbi)

```

Comment:
We have 2 uses of `calculation.sbi` here

To make it consistent w/ line 8 we could define `sbi` as `calculation.sbi`

However, I would say that I would possibly prefer reverting lines 8-10 to this format were line 8 disappears and in the params we explicitly say `settlement.paymentRequestId`

Reason being is `paymentRequestId` is a field stored in _maaaaanny_ tables so knowing which table's `paymentRequestId` we're referring to might be v important

Whatcha think?